src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,15 +2,20 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
         //TODO: add corresponding log message about method login was called
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("login methode was called, login: {}", user);
```

Comment:
There is a typo in the log message: 'methode' should be 'method'. Additionally, consider logging the login parameter instead of the user object for clarity, like this: `logger.info("login method was called, login: {}", login);`.